lib/bots/versionbot.ts

```diff
@@ -769,9 +769,9 @@ export class VersionBot extends ProcBot {
         };
 
         return Promise.mapSeries([
-            `git clone https://${versionData.authToken}:${versionData.authToken}@github.com/` +
-                `${versionData.repoFullName} ${versionData.fullPath}`,
-            `git checkout ${versionData.branchName}`
+            `git clone "https://${versionData.authToken}:${versionData.authToken}@github.com/` +
+                `${versionData.repoFullName}" ${versionData.fullPath}`,
+            `git checkout "${versionData.branchName}"`
```

Comment:
We need to do some much more thorough escaping than this. I think branch names can include nearly anything, definitely including double quotes (which will definitely break this escaping), and we're passing this string directly to the CLI!

The one thing that's holding back branches like `"; rm -rf / #` from being completely viable is that you're not allowed spaces, but I suspect there's still probably a fair few crazy breaking things you can come up with here. `";shutdown` as a branch name will cause some havoc, for starters.